spec/widgets/category/category-widget-model.spec.js

```diff
@@ -32,6 +33,23 @@ describe('widgets/category/category-widget-model', function () {
     });
   });
 
+  describe('search state', function () {
```

Comment:
Well, take into account that we are checking that [this change](https://github.com/CartoDB/deep-insights.js/pull/151/files#diff-9174d99f35e6d799c1c69c256b44b3c3R126) is working, and it is in the `category-widget-model.js`.